source/release-notes/1.4.txt

```diff
@@ -27,7 +25,50 @@ Starting in 1.4.0, you can set the workload level that
 destination clusters. For more information, see the
 :option:`--loadLevel` option.
 
+Improved Resiliency
+~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` 1.4.0 improves :term:`oplog` handling. The ``oplog``
+handling improvement adds resiliency to long running migrations.
+
+Faster Processing for Sharded Clusters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Version 1.4.0 improves performance for replica sets and single-shard
+clusters. Multi-shard clusters have even better speed improvements.
+
+Telemetry
+~~~~~~~~~
+
+Starting in 1.4.0, ``mongosync`` collects anonymous, aggregated usage
+data to improve MongoDB products. You can disable collection of this
+:ref:`telemetry <c2c-telemetry>` data when you start ``mongosync``.
+
+Bug Fixes
+----------
+
+- Fixes an issue so documents with ``_id`` values that evaluate to the
+  same string replicate correctly.
+
+- Fixes a collection naming issue. Prior to version 1.4.0, ``mongosync``
+  crashes when a new collection reuses a dropped collection's name with
+  letter case changes.
+  
+  For example, previous versions of ``mongosync`` crash if a new
+  ``SampleName`` collection replaces the dropped ``samplename``
+  collection.
+
+Additional Considerations
+-------------------------
+
+The MongoDB server's :dbcommand:`validate` command may report data
+corruption on destination clusters that run MongoDB version 6.0 through
+6.0.6. The data on the destination server is not corrupt. A bug in the
+:ref:`WiredTiger <storage-wiredtiger>` database engine,
+:issue:`WT-11051`, causes the false report.
+
```

Comment:
Apologies for the late EOD comment -- I just wrapped up [REP-2770](https://jira.mongodb.org/browse/REP-2770) and think we may want to add something in here for that. The ticket's Release Notes describe the exact change, but we can also keep it high-level and say something like:

> Mongosync uses less memory for buffering change events, alleviating some out-of-memory issues.

(@mmcclimon if you agree / have thoughts)

Reply:
Yeah, I agree it would be good to put something here; it's a change end users will definitely notice!